web/package-lock.json

```diff
@@ -0,0 +1,6546 @@
+{
```

Comment:
@bendichter why that package-lock.json is getting introduced here at all, or why is it becoming needed for adding this feature?

Reply:
isn't this a file that gets automatically generated? Maybe it should be in .gitignore?

Reply:
This file is generated by `npm` whereas we have always used `yarn` to build the frontend. I'm planning to revert the addition of this file, along with the changes in yarn.lock, as I don't believe they are needed for this PR. (You can do this yourself, @bendichter, if you prefer, but it's something I can easily do when I work on this.)

Reply:
oh, I see, this was created when I accidentally ran nmp install instead of yarn install.